src/components/MeetInfo-Table/MeetInfoTable.style.js

```diff
@@ -35,7 +35,7 @@ export const TableWrapper = styled.div`
 			}
 		}
 		th {
-			background-color: ${props => props.theme.grey0E0E0EToGreen3C494F};
+			background-color: ${props => props.theme.DarkTheme ? "#05A794" : props.theme.grey0E0E0EToGreen3C494F};
```

Comment:
hey @hirentimbadiya,
I've recently started contributing to layer repos, so feel free to correct/improvise me if I'm wrong.   I think it'd be better to store the color hex string in the themes instead of using it directly here.

Reply:
@randychilau might be able to offer some counsel here.

Reply:
Hi @hirentimbadiya,

Thanks for working on this issue!

We try to stay away from using conditionals for dark mode because if a user has a darkMode preference in their system and loads the page in their browser, the elements with conditionals will flash or flicker from light mode to dark mode (this also happens when a page in dark mode is reloaded in the browser).

![](https://user-images.githubusercontent.com/90356410/225827119-0c7967ed-2c7b-4271-9c38-aa0a6d50b8e3.gif)

The preferred method is to not use a conditional as [explained here](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-dark-mode-for-components) and try to use an existing color pair in the [themeStyles.js file](https://github.com/layer5io/layer5/blob/master/src/theme/app/themeStyles.js) such as `props.theme.grey0E0E0EToGreen3C494F` or `props.theme.whiteToBlack`.

If you have to create a new color pair, then you can follow the [instructions here](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-a-new-color-pair-transition-property) and use that new property in your code.

Happy coding and feel free to ask me any questions.

Thanks,
Randy